metau-capstone/StudyViewController.m

```diff
@@ -31,9 +33,35 @@ @implementation StudyViewController
 - (void)viewDidLoad {
     [super viewDidLoad];
     PFUser *user = [PFUser currentUser];
-    // Construct Query
     
-    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"(userID = %@) AND ((levelNum = 1) OR (levelNum = 5))", user.objectId];
+    // Instantiate flashcard sides
```

Comment:
Kudos for pulling this out!

This is probably a good example of something you could put in an instantiateCards method to not entirely bloat the view did load and break out the operations.